### DISCUSSION TITLE
New install on new machine, will the teslamate restore restore a functional system

### DISCUSSION THREAD

**Discussion by M-ax-G** [teslamate-org/teslamate#3009] [Q&A] [answered]
I have a broken system, where after an update of Telsamate from 1.26 to 1.27; the teslamate container no longer stays up.

I have installed a new machine; put docker on it, pulled teslamate via the docker-compose.yml file.

I have not logged into TeslaMate, where is is asking for API keys and token.

I have created the teslamate backup as per https://docs.teslamate.org/docs/maintenance/backup_restore

I could restore it, but before I do and bugger something up I was wondering what the correct approach is?

I have my old key and refresh token, but the token says it expires in Dec 2021.

Is restoring the teslamate.bck sufficient to bring the whole system back? (With access token, key etc. and what else is required to make this work again)?
Or is there anything else I have to do?

**Comment by DrMichael** [comment, marked as answer]
Usually just restoring the backup should be ok.

**Reply by M-ax-G** [reply]
Wow; it does work :)

I connected to :4000 and had straight access to the vehicle status.
And :3000 is showing me what's going on.
Nice... thank you.
(I am so glad this thing is back up and running again.)
